README.md

```diff
@@ -22,6 +23,14 @@ Xamarin.Android provides open-source bindings of the Android SDK for use with
 
 ## Current
 
+`Android for .NET` ships as a workload through the `dotnet` workload system in [.NET 6+][dotnet-download]. See
+the [workload documentation][workload-documentation] for installation commands.
+
+[dotnet-download]: https://dotnet.microsoft.com/en-us/download
+[workload-documentation]: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-workload-install
+
+Legacy `Xamarin.Android` installers are available here:
```

Comment:
I wouldn't use "legacy", maybe do:
```suggestion
Classic `Xamarin.Android` installers are available here:
```

Reply:
👍 I went back and forth on this, so thanks for the feedback.  Changed to `classic`.